Dockerfile.local

```diff
@@ -2,14 +2,6 @@ FROM joplin-base
 
 ENV DEPLOYMENT_MODE "LOCAL"
 
-
-RUN apt-get update \
```

Comment:
Why don't we get the updated postgres here anymore? I'm sure I'm just missing something.

Reply:
It's already installed in Dockerfile.base, which is used here at "FROM joplin-base"